src/react-components/src/components/03-organisms/Hero/Hero.stories.tsx

```diff
@@ -0,0 +1,90 @@
+import * as React from "react";
+
+import Image from "../../01-atoms/Images/Image/Image";
+import Heading from "../../01-atoms/Text/Headings/Heading";
+import { HeroTypes } from "./HeroTypes";
+import Hero from "./Hero";
+import SectionName from "../../00-base/SectionName";
+import Placeholder from "../../00-base/Placeholder";
+
+export default {
+  title: "Hero",
+  component: Hero,
+};
+
+export const heroPrimary = () => <Hero
+  heroType={HeroTypes.Primary}
+  heroId="2"
+  heading={<Heading 
+    level={1} 
+    id={"1"} 
+    text={"Hero Primary"}
+    blockName={"hero"}
```

Comment:
Is it required for the `Heading` to have a blockName of `"hero"`? If that's the case, is it written down somewhere? Also, what happens if you forget to include this prop in `Heading`?

Reply:
It is required, and it would fail as a result. How would something like this typically be handled?

Reply:
if all heros should have blockname "hero", I would figure out how to incorporate it into the component itself rather than expecting users to to remember to pass it.

Reply:
Talked about this with @hokei over Slack. Handling this particular problem (how sub-JSX.Elements get the correct blockNames within larger components) is out of scope for this ticket, so I'm going to leave this as-is for now.